resources/web/wwi/FloatingRobotWindow.js

```diff
@@ -0,0 +1,131 @@
+        this.floatingRobotWindowContent.className = 'floating-robotWindow-content';
+        this.floatingRobotWindow.appendChild(this.floatingRobotWindowContent);
+
+        let robotWindow = document.createElement('iframe');
+        robotWindow.id = this.name+"-robotWindow";
+        robotWindow.src = this.url+"/robot_windows/"+this.window+"/"+this.window+".html?name="+this.name;
+        this.floatingRobotWindowContent.appendChild(robotWindow);
+
+        this.dragElement(document.getElementById(name));
+    }
+
+    getID() {
+        return this.floatingRobotWindow.id;
+    }
+
+    setSize(w, h) {
+        this.floatingRobotWindow.style.width = w.toString()+'px';
+        this.floatingRobotWindow.style.height = h.toString()+'px';
+    }
+
+    setPosition(xPos, yPos) {
+        this.floatingRobotWindow.style.left = xPos.toString()+'px';
+        this.floatingRobotWindow.style.top = yPos.toString()+'px';
+    }
+
+    changeVisibility() {
+        if (this.floatingRobotWindow.style.visibility == 'hidden') 
+            this.floatingRobotWindow.style.visibility = 'visible';
+        else
+            this.floatingRobotWindow.style.visibility = 'hidden';
+    }
+
+    setVisibility(visibility) {
+        this.floatingRobotWindow.style.visibility = visibility;
+    }
+
+    getVisibility() {
+        return this.floatingRobotWindow.style.visibility;
+    }
+
+    dragElement(rw) {
+        var pos1 = 0, pos2 = 0, pos3 = 0, pos4 = 0;
+        var pageHeight = rw.parentNode.parentNode.parentNode.offsetHeight;
+        var containerHeight = rw.parentNode.parentNode.offsetHeight;
+        var containerWidth = rw.parentNode.parentNode.offsetWidth;
+        var topOffset = 0, leftOffset = 0;
+
+        rw.firstChild.onmousedown = dragMouseDown;
+        rw.addEventListener('resize', function(event) {console.log("Risizing "+event.srcId);});
+
+        function dragMouseDown(event) {
+            rw.lastElementChild.style.pointerEvents = 'none';
+
+            pageHeight = rw.parentNode.parentNode.parentNode.offsetHeight;
+            containerHeight = rw.parentNode.parentNode.offsetHeight;
+            containerWidth = rw.parentNode.parentNode.offsetWidth;
+
+            event.preventDefault();
+
+            pos1 = event.clientX;
+            pos2 = event.clientY;
+            topOffset = pos2 - rw.offsetTop
+            leftOffset = pos1 - rw.offsetLeft;
+
+            document.onmouseup = closeDragElement;
+            document.onmousemove = robotWindowDrag;
+        }
+
+        function robotWindowDrag(event) {
+            event.preventDefault();
+
+            pos3 = pos1 - event.clientX;
+            pos4 = pos2 - event.clientY;
+            pos1 = event.clientX;
+            pos2 = event.clientY;
+
+            let top = rw.offsetTop - pos4;
+            let left = rw.offsetLeft - pos3
+            let bottom = top + rw.offsetHeight;
+            let right = left + rw.offsetWidth;
+
+            if (top<0 || event.clientY<topOffset)
```

Comment:
```suggestion
            if (top < 0 || event.clientY < topOffset)
```
This happens at other places as well.